dolphinscheduler-ui/src/views/projects/workflow/instance/gantt/components/gantt-chart.tsx

```diff
@@ -80,7 +80,7 @@ const GanttChart = defineComponent({
       minTime = minTime < start ? minTime : start
       maxTime = maxTime > end ? maxTime : end
       data[task.status].push({
-        name: task.name,
+        name: task.taskName,
```

Comment:
You should not change it. Because it is converted from `taskName` to `name` by function `formatGantt`.
<img width="508" alt="image" src="https://user-images.githubusercontent.com/8847400/187106883-eacd27bc-6ff9-44ee-868a-c68b0b127742.png">

Reply:
Thanks for reminding. IMHO we can remove `name` here since it's just a dup of taskName?
<img width="438" alt="Screen Shot 2022-08-29 at 10 35 47" src="https://user-images.githubusercontent.com/49837965/187111825-7d35f09e-f6c3-4da2-b2fd-61cfb4c741a6.png">

Reply:
> Thanks for reminding. IMHO we can remove `name` here since it's just a dup of taskName? <img alt="Screen Shot 2022-08-29 at 10 35 47" width="438" src="https://user-images.githubusercontent.com/49837965/187111825-7d35f09e-f6c3-4da2-b2fd-61cfb4c741a6.png">

sure, you can optimize it.